### PR TITLE
DOCSP-28301-update-to-driver

### PR DESCRIPTION
**Pull request by davemungo**
STAGING
[Considerations](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-28301-update-to-driver/write-scripts/considerations/)
[Release notes](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-28301-update-to-driver/changelog/)
[Data types](https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-28301-update-to-driver/reference/data-types/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-28301)
[MONGOSH] Update to driver and bson v5.x

[BUILD 2023-03-13](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=640f3943054271fb0a56d309)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by ianf-mongodb — APPROVED** [review]
@davemungo This one LGTM Dave!

**Comment by davemungo** [comment]
@addaleax when you have a moment would you have a look at this PR? Thanks

File: source/write-scripts/considerations.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-call-limitations.rst
+
+Code Objects
+~~~~~~~~~~~~
+
+.. include:: /includes/scripting/fact-code-objects.rst
```
Comment (addaleax) [inline source/write-scripts/considerations.txt:21]: I’m not sure I really get why these things are being listed here. The `Code` object note in particular is very specific to usage of the `Code()` class and not related to scripting. (I don’t think we even document `Code()` explicitly?)

**Review by davemungo — COMMENTED** [review]
@addaleax  thanks for the feedback, this is ready for another look

**Review by addaleax — APPROVED** [review]

**Pull request merged**